architecture/core_objects/builds.adoc

```diff
@@ -171,55 +171,12 @@ You can define multiple Docker registry entries in this file. Alternatively, you
 can also add authentication entries to this file by running the `docker login`
 command. The file will be created if it does not exist.
 
-Kubernetes provides the
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords. You must
-first create the `*Secret*` before builds can use your *_.dockercfg_* file for
-pushing the output image:
-
-====
-----
-$ osc create -f secret.json
-----
-====
-
-The `*data*` field for the `*Secret*` object must contain the `*dockercfg*` key
-with the value set to the base64-encoded content of the *_.dockercfg_* file. For
```

Comment:
https://github.com/openshift/origin/pull/1453 is what I was thinking about, but from looking now at some auto-generated secrets like `builder-dockercfg*` and `default-dockercfg*` (which have `.dockercfg` in `data`), I'm even less sure.